test/spec/similarity_spec.rb

```diff
@@ -87,6 +87,29 @@
     expect(@driver.find_elements(:css, '.media__relationship').size).to eq 1
   end
 
+  it 'should create standalone fact check', bin7: true do
```

Comment:
I think this test is more about testing the relevant items feature than standalone fact checks, so maybe should be named that?  Or if we need to test the standalone fact check helper function explicitly, we should separate into two tests?